flow-typed/messages.json.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+declare module "./messages.json" {
+    // TODO: Update to use a regexp once https://github.com/facebook/flow/issues/4654 is resolved.
+    declare type MessageKey = string;
+
+    declare type SingleLanguageBundle = {
+        [MessageKey]: string
+    };
+
+    // TODO: Figure out a better way to handle this, this is the only way I can keep it from complaining about using messages.en in another file.
```

Comment:
I'd like to try keeping line lengths shorter than this. Perferrably to 80 characters if possible. I don't think I want to impose a hard limit and rather use 80 characters as a suggestion, if it's possible/feasible in a given situation. Sometimes it isn't possible or will make the code less clear. But in cases where it doesn't hurt, I'd like to limit the line lengths. For TODO comments, we have been indenting to match the first line. So, this comment would be formatted as:

```
    // TODO: Figure out a better way to handle this, this is the only way I can
    //       keep it from complaining about using messages.en in another file.
```

Reply:
So, interestingly, the CRA build process turns any warnings into hard errors, and by default the build process [uses its internal ESLint rules](https://create-react-app.dev/docs/setting-up-your-editor/#displaying-lint-output-in-the-editor).   Even with `ESLINT_EXTEND` set, I couldn't get it to stop complaining about the code lines.

As a temporary solution, I added ESLint hints for overly long lines of code.